### ISSUE TITLE
No available wallet error while creating an account in v1.1.0

### ISSUE THREAD

**Issue by HomoEfficio** [EOSIO/eos#4784]
This is what I did to create an account.

1. Created a wallet named tmp
    ```
    $ cleos --wallet-url=http://localhost:8899 wallet create -n tmp
    Creating wallet: tmp
    Save password to use in the future to unlock this wallet.
    Without password imported keys will not be retrievable.
    "PW***"
    ```

1. Imported private key of eosio into the wallet
    ```
    $ cleos --wallet-url=http://localhost:8899 wallet import -n tmp --private-key
    private key: 
    imported private key for: EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV
    ```

1. Created 2 keys
    ```
    $ cleos create key
    Private key: 5***
    Public key: EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md

    $ cleos create key
    Private key: 5***
    Public key: EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5
    ```

1. Checked if the wallet is unlocked
    ```
    $ cleos --wallet-url=http://localhost:8899 wallet list
    Wallets:
    [
      "tmp *"
    ]
    ```

1. Created an account named tmp and got error
    ```
    $ cleos create account eosio tmp EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5
    Error 3120006: No available wallet
    Ensure that you have created a wallet and have it open
    ```

1. Opened the wallet and list the wallet
    ```
    $ cleos --wallet-url=http://localhost:8899 wallet open -n tmp
    Opened: tmp

    $ cleos --wallet-url=http://localhost:8899 wallet list
    Wallets:
    [
      "tmp"
    ]
    ```

1. Created an account again and got error again
    ```
    $ cleos create account eosio tmp EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5
    Error 3120006: No available wallet
    Ensure that you have created a wallet and have it open
    ```

1. Unlock the wallet again and check if it is unlocked
    ```
    $ cleos --wallet-url=http://localhost:8899 wallet unlock -n tmp
    password: 
    Unlocked: tmp

    $ cleos --wallet-url=http://localhost:8899 wallet listWallets:
    [
      "tmp *"
    ]
    ```

1. Created an account again and got error again
    ```
    $ cleos create account eosio tmp EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5
    Error 3120006: No available wallet
    Ensure that you have created a wallet and have it open
    ```

In summary, I made a wallet into which the private key of the creator(eosio) is imported and tried to create an account with creator(eosio) and 2 public keys. It was successful in v1.0.10 but I could not make an account in the same way in v1.1.0. So how can I create an account?

**Comment by andriantolie** [comment]
It's because in your command 
`cleos create account eosio tmp EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5`
You don't specify the wallet url. I you don't specify the wallet url, it will default to `--wallet-url=http://localhost:8900` which is different from the wallet that you have configured above.

Try specifying the wallet url to port 8899 in your create account command
`cleos --wallet-url=http://localhost:8899 create account eosio tmp EOS8NVsqJuT7Anxd3gPSe7HWsYnB5mDdMxSi4SQKnQdbvak3q23Md EOS6UvXZSbChoFHxxvVR1D81oRXxpWR5zLNQyeSntyVjxWSP5Aem5`
